iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -29,6 +29,10 @@ static llvm::cl::opt<unsigned> pipelineDepth("iree-codegen-cuda-pipeline-depth",
                                              llvm::cl::desc("Pipeline depth"),
                                              llvm::cl::init(4));
 
+static llvm::cl::opt<unsigned> logSwizzleTile(
+    "iree-codegen-cuda-log-swizzle-tile",
```

Comment:
Please rename to `iree-codegen-log-swizzle-tile`.

Reply:
Done.